coins/src/adapters/tokenMapping_added.json

```diff
@@ -4897,16 +4897,16 @@
       "to": "coingecko#wrapped-steth"
     },
     "0x04619e9ce4109590219c5263787050726be63382148538f3f936c22aa87d2fc2": {
-      "name": "Nostra Staked STRKr",
+      "name": "Nostra Staked STRK",
       "decimals": "18",
       "symbol": "nstSTRK",
-      "to": "coingecko#starknet"
+      "to": "coingecko#staked-strk"
     },
     "0x4619e9ce4109590219c5263787050726be63382148538f3f936c22aa87d2fc2": {
-      "name": "Nostra Staked STRKr",
+      "name": "Nostra Staked STRK",
       "decimals": "18",
       "symbol": "nstSTRK",
-      "to": "coingecko#starknet"
+      "to": "coingecko#staked-strk"
```

Comment:
these 2 mappings arent used, we use coins/src/adapters/other/nstSTRK.ts

Reply:
I think this adapter should be removed since we have Coingecko listing for nstSTRK.